go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/beevik/ntp v0.3.0
 	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/dennwc/btrfs v0.0.0-20230312211831-a1f570bd01a1
+	github.com/discordianfish/go-runit v0.0.0-20150630195641-06ad41a06c4a
```

Comment:
might need a `go mod tidy`, github.com/soundcloud/go-runit is still listed below and in go.sum